### PR TITLE
Upgraded VCN module to version 3.0.0

### PR DESCRIPTION
**Pull request by hyder**
- Upgraded VCN module to 3.0.0
- Upgraded bastion module to rc3
- Upgraded operator module to rc6

### CODE REVIEW

File: terraform.tfvars.example [hunk 1]
```diff
@@ -119,7 +103,7 @@ subnets = {
   workers  = { netnum = 1, newbits = 2 }
 }
 
-vcn_cidr      = "10.0.0.0/16"
+vcn_cidr      = ["10.0.0.0/16"]
```
Comment (kral2) [inline terraform.tfvars.example:106]: this should be plural form: `vcn_cidrs`

**Review by kral2 — CHANGES_REQUESTED** [review]
beside the variable naming error on terraform.tfvars.example, lgtm.

File: main.tf [hunk 2]
```diff
@@ -3,32 +3,41 @@
...
+
+  # drg
+  create_drg       = var.create_drg
+  drg_display_name = var.drg_display_name
+
+  # lpgs
+  local_peering_gateways = var.local_peering_gateways
+
+  # tags
+  freeform_tags = var.tags["vcn"]
```
Comment (karthicgit) [inline main.tf:26]: lets keep the freeform_tags naming convention across the modules.

**Comment by hyder** [comment]
Closes #360

File: main.tf (hunk 2 above)
Comment (hyder) [reply main.tf:26]: This PR is for VCN 3.0.0. Can you please open a new issue for the freeform_tags?

**Review by kral2 — APPROVED** [review]

**Review by karthicgit — APPROVED** [review]

**Pull request merged**